python-sdk/tests/sql/test_table.py

```diff
@@ -137,8 +137,8 @@ def test_table_to_datasets_extra():
     "table",
     [
         Table(),
-        Table("_tmp"),
-        Table(name="_tmp", conn_id="test_conn"),
+        Table("_tmp_a"),
+        Table(name="_tmp_a", conn_id="test_conn"),
```

Comment:
@ashb Why did we change these names?

Reply:
We had inconsistency in `_tmp_` (defined by `TEMP_PREFIX`) and what we checked for in Table's `__init__` as shown in the code-blocks below

https://github.com/astronomer/astro-sdk/blob/5d714ad468e3f8ddb4f14741b6385d0d3f89b85d/python-sdk/src/astro/table.py#L16

https://github.com/astronomer/astro-sdk/blob/0e9d07b53bb2218dabecdbb991dd49cc2ff94cfe/python-sdk/src/astro/table.py#L72

In general, users are “NOT” passing names at all for temp tables, so I had suggested to just make it consistent and use the value in `TEMP_PREFIX` because that is what we use for generating table names. (probably >99.5% cases)

Reply:
And I've now reverted this change (or rather force pushed without this commit) in favour of the next commit.